docker-images/cross-build/README.md

```diff
@@ -16,18 +17,23 @@ LATEST_GOLANG=$(curl -sSfL "https://go.dev/VERSION?m=text" | /bin/grep -Po '^go(
 # build a container image named "zitibuilder" with the Dockerfile in the top-level of this repo
 docker build \
     --tag=zitibuilder \
-    --file=Dockerfile.linux-build \
-    --build-arg latest_golang=${LATEST_GOLANG} \
     --build-arg uid=$UID \
-    --build-arg gid=$GID .
+    --build-arg gid=$GID \
+    ./docker-images/cross-build/
 ```
 
+<!--
+    I pruned this arg because it doesn't currently match the go.dev URL, and
```

Comment:
this comment is strange to see here? what's this about?

Reply:
I found a way to solve the issue I was commenting about, so I removed the comment.